### PR TITLE
Eth Plugin: Update Instance Snapshot

### PR DESCRIPTION
**Pull request by topocount**
The snapshot tests for the instance now verify that the Identity load
logic is behaving correctly in the eth plugin

test plan: sharness snapshot tests should pass

### CODE REVIEW

File: sharness/__snapshots__/test-instance/config/plugins/sourcecred/ethereum/ethereumAddresses.json [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"type":"sourcecred/ethAddressFile","version":"0.0.1"},["0x2Ccc7cD913677553766873483ed9eEDdB77A0Bb0", "0xb4124cEB3451635DAcedd11767f004d8a28c6eE7","0x8401Eb5ff34cc943f096A32EF3d5113FEbE8D4Eb"]]
```
Comment (wchargin) [inline sharness/__snapshots__/test-instance/config/plugins/sourcecred/ethereum/ethereumAddresses.json:1]: As mentioned, you can pass `, {space: 4}` to `json-stable-stringify` to
get it to spit out a diff-friendly format.

File: sharness/__snapshots__/test-instance/output/graphs/sourcecred/ethereum/graph.json [hunk 2]
```diff
@@ -0,0 +1 @@
+[{"type":"sourcecred/weightedGraph","version":"0.1.0"},{"graphJSON":[{"type":"sourcecred/graph","version":"0.8.0"},{"edges":[],"nodes":[],"sortedNodeAddresses":[]}],"weightsJSON":[{"type":"sourcecred/weights","version":"0.2.0"},{"edgeWeights":{},"nodeWeights":{}}]}]
```
Comment (decentralion) [inline sharness/__snapshots__/test-instance/output/graphs/sourcecred/ethereum/graph.json:1]: A human readable graph.json... that's a first

**Review by decentralion — COMMENTED** [review]
I think this PR needs to be rebased

**Comment by topocount** [comment]
@decentralion @wchargin this has been rebased and modified. Please re-review when you have the chance.

**Review by decentralion — APPROVED** [review]

**Pull request merged**